source/common/network/address_impl.cc

```diff
@@ -345,15 +351,19 @@ PipeInstance::PipeInstance(const sockaddr_un* address, socklen_t ss_len)
   }
   address_ = *address;
   if (abstract_namespace_) {
+    if (mode != 0) {
+      throw EnvoyException("Cannot set mode for Abstract AF_UNIX sockets");
```

Comment:
test coverage for this?

Reply:
[Added a test](https://github.com/envoyproxy/envoy/pull/8423/files#diff-a97b9874a5d6e4ba7ea09da95a425b58R346) to cover this